packages/Card/src/Card.styles.js

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components";
+import tokens from "@paprika/tokens";
+import { ShirtSizes } from "@paprika/helpers/lib/customPropTypes";
+
+// Common
+
+const commonStyles = styled.div`
+  background: ${tokens.color.white};
+  border: 1px solid #f0f0f0;
+  border-radius: ${tokens.card.borderRadius};
+  box-shadow: ${tokens.shadow};
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
+  position: relative;
+`;
+
+// Sizes
+
+const sizeStyles = {
+  [ShirtSizes.SMALL]: `
+    width: 300px;
+    height: 130px;
+  `,
+  [ShirtSizes.MEDIUM]: `
+  width: 300px;
+  height: 320px;
+  `,
+  [ShirtSizes.LARGE]: `
+  width: 300px;
+  height: 500px;
+  `,
+};
+
+// Modifiers
+
+const fullWidthStyles = `
+  display: flex;
+  width: 100%;
+`;
+
+const autoHeightStyles = `
+  display: flex;
+  height: auto;
+`;
+
+// States
+
+const disabledStyles = `
+  box-shadow: none;
+  color: ${tokens.color.blackDisabled};
+  cursor: not-allowed;
+  text-decoration: none;
+  text-shadow: none;
+
+  &:focus {
```

Comment:
🤔 why do we have two focus are we overriding something? I think @oscarkwan might have a better understanding for this.

Reply:
I think they are just trying to extract out the styles. But yeah I would also like to know more on what you are trying to achieve here. I couldn't find a story with a disabled card style so i couldn't really test this part